### PR TITLE
[NL Next] Add overview classifier

### PR DESCRIPTION
**Pull request by juliawu**
This PR:
* Adds an "overview" classifier to look for "tell me about <place>" and "more about <place>" queries
* Hooks the classifier up to debug info
* Adds unit-tests for the new classifier

Example:
image

### CODE REVIEW

File: server/lib/nl/constants.py [hunk 1]
```diff
@@ -263,6 +263,7 @@
                 "loss",
             ],
         },
+        "Overview": ["(tell me|more) about",],
```
Comment (pradh) [inline server/lib/nl/constants.py:266]: lets just stick to "tell me about" per guha's last comment?

File: server/lib/nl/detection.py [hunk 2]
```diff
@@ -231,6 +238,7 @@ class ClassificationType(IntEnum):
     ClassificationType.CORRELATION,
     ClassificationType.TIME_DELTA,
     ClassificationType.EVENT,
+    ClassificationType.OVERVIEW,
```
Comment (pradh) [inline server/lib/nl/detection.py:241]: Can you remove from this?  The ranking here I think could result in not fulfilling the query when we run with this change

Comment (pradh) [reply server/lib/nl/detection.py:241]: (might be worth testing what happens with the query)

**Review by pradh — COMMENTED** [review]
Thanks for the quick PR!

File: server/lib/nl/constants.py (hunk 1 above)
Comment (juliawu) [reply server/lib/nl/constants.py]: Done!

File: server/lib/nl/detection.py (hunk 2 above)
Comment (juliawu) [reply server/lib/nl/detection.py]: Removed! Tested and "tell me about tulare county" and "tell me about palo alto" work after removal.

**Review by pradh — APPROVED** [review]

**Pull request merged**